widgetssdk/build.gradle

```diff
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 31
+    compileSdkVersion 33
     buildToolsVersion "30.0.3"
     defaultPublishConfig "debug"
     defaultConfig {
         minSdkVersion 24
-        targetSdkVersion 31
+        targetSdkVersion 33
```

Comment:
If I'm not mistaken, targetSdkVersion increase may cause issues for the integrators and we decided to do this as later as possible, don't we?

Reply:
in core SDK targetSdkVersion is 32, I remember that we had such problem, but it seems we didn't downgrade version.